ivadomed/loader/bids_dataframe.py

```diff
@@ -94,8 +94,8 @@ def create_bids_dataframe(self):
                     subject_path = path_object.parts[subject_path_index]
                     if path_object.name == "samples.tsv" or path_object.name == "samples.json":
                         force_index.append(path_object.name)
-                    if (path_object.name.endswith(ext_microscopy) and (path_object.parent.name == "microscopy" or
-                            path_object.parent.name == "micr") and subject_path.startswith('sub')):
+                    if (path_object.name.endswith(ext_microscopy) and path_object.parent.name == "micr" and
```

Comment:
Just wondering. Given now that "micr" is standarized in BIDS, should we actually create a keyword for this? I am not sure if we have a relevant class already.

Reply:
I don't think we need to create keywords for this. It is a BIDS "data type" like any other. For example in MRI, there are several data types (ex: `anat`, `dwi`, `func`). We normally don't see them in our ivadomed code because it's pybids and not ivadomed that deals with them to index the files.
Again, in the microscopy case, we see "micr" in the code because we need to force-index the files as microscopy in not integrated in pybids yet.